packages/components/primary-action-dropdown/src/primary-action-dropdown.stories.tsx

```diff
@@ -0,0 +1,71 @@
+import type { ComponentProps } from 'react';
+import type { Meta, StoryObj } from '@storybook/react';
+import { BoxIcon, BrainIcon, FlameIcon } from '@commercetools-uikit/icons';
+import PrimaryActionDropdown from './primary-action-dropdown';
+import Option from './option';
+
+type PrimaryActionDropdownPropsAndCustomArgs = ComponentProps<
+  typeof PrimaryActionDropdown
+> & {
+  isDisabledOption1: boolean;
+  isDisabledOption2: boolean;
+  isDisabledOption3: boolean;
+  labelOption1: string;
+  labelOption2: string;
+  labelOption3: string;
+};
+
+const meta = {
+  title: 'Components/PrimaryActionDropdown',
+  component: PrimaryActionDropdown,
+  tags: ['autodocs'],
+  parameters: {
+    docs: {
+      story: {
+        inline: false,
+        iframeHeight: 180,
+      },
+    },
+  },
+} satisfies Meta<PrimaryActionDropdownPropsAndCustomArgs>;
+
+export default meta;
+type Story = StoryObj<PrimaryActionDropdownPropsAndCustomArgs>;
+
+export const Default: Story = {
+  args: {
+    isDisabledOption1: false,
```

Comment:
We can also group this one to differentiate them from the actual component props.

Reply:
Sometimes I think a little repetition makes code easier to understand and thus, to maintain.

What do you think about something like this instead?
```js
   const categorize = (index: number) => ({ table: { subcategory: `Option ${i}` } });
  
  ...

    isDisabledOption1: categorize(1),
    isDisabledOption2: categorize(2),
    isDisabledOption3: categorize(3),
    labelOption1: categorize(1),
    labelOption1: categorize(2),
    labelOption1: categorize(3),
```

Reply:
Sure, let's focus on getting the first PR merged and then I'll take care of the rest after rebasing.